plugin/entryAttributes.go

```diff
@@ -251,11 +258,13 @@ func (a *EntryAttributes) UnmarshalJSON(data []byte) error {
 		a.SetCtime(t)
 	}
 	if mode, ok := mp["mode"]; ok {
-		m, err := munge.ToFileMode(mode)
-		if err != nil {
-			return attrMungeError("mode", err)
+		// Without a type hint, all numbers are given as floats. This represents the uint32 type
```

Comment:
Is this basically saying "Even though os.FileModes are uint32 types, json.Unmarshal unmarshals them as float64. That's ok, because float64 has sufficient precision to represent all uint32 types." ? If so, the current comment could be clearer because I had to think a little bit about what you meant here.